scripts/rook.sh

```diff
@@ -82,6 +82,14 @@ kubectl_retry() {
 
 function deploy_rook() {
         kubectl_retry create -f "${ROOK_URL}/common.yaml"
+
+        # If rook version is > 1.5 , we will apply CRDs.
+        ROOK_MAJOR=$(rook_version 1)
+        ROOK_MINOR=$(rook_version 2)
+        if  { [ "${ROOK_MAJOR}" -eq 1 ] && [ "${ROOK_MINOR}" -ge 5 ]; }
```

Comment:
why the extra `{ .. }`, we do not use that in other places either

Reply:
no harm. but removed.